types/seat/wlr_seat_pointer.c

```diff
@@ -241,6 +241,9 @@ void wlr_seat_pointer_send_axis(struct wlr_seat *wlr_seat, uint32_t time,
 
 		uint32_t version = wl_resource_get_version(resource);
 
+		if (version >= WL_POINTER_AXIS_SOURCE_SINCE_VERSION) {
+			wl_pointer_send_axis_source(resource, source);
```

Comment:
Is our axis source enum guaranteed to be compatible with the wayland one?

Reply:
Yeah, just like the orientation one.